airflow/models/taskinstance.py

```diff
@@ -1604,6 +1604,37 @@ def render_templates(self, context: Optional[Context] = None) -> None:
 
         self.task.render_template_fields(context)
 
+    def render_k8s_pod_yaml(self) -> Optional[str]:
+        """Render k8s pod yaml"""
+        # if conf.get("core", "executor") != "KubernetesExecutor":
```

Comment:
@dimberman did you intend on adding this back or should I just remove the comment? FWIW, I've added similar logic in the view that will 404 if the executor is not `KubernetesExecutor` or `CeleryKubernetesExecutor`. I would guess that's sufficient?

Reply:
@ryanahamilton yeah that sounds sufficient!